src/main/java/org/hl7/davinci/priorauth/ClaimEndpoint.java

```diff
@@ -756,7 +758,7 @@ public void run() {
             } else {
               logger.severe("SubscriptionHandler::Unable to send web-socket notification for subscription "
                   + subscriptionId + " because web-socket id is null. Client did not bind a websocket to id");
-              App.getDB().update(Database.SUBSCRIPTION, Collections.singletonMap("id", subscriptionId),
+              App.getDB().update(Table.SUBSCRIPTION, Collections.singletonMap("id", subscriptionId),
                   Collections.singletonMap("status", SubscriptionStatus.ERROR.getDisplay().toLowerCase()));
             }
           }
```

Comment:
700 lines is getting long for a single file.    Possibly refactor in future if there is common code.

Reply:
yea I agree. This file has always been really long since it holds the majority of the logic. Refactoring doesn't seem trivial so I will add it to the backlog